doc/release/0.3.rst

```diff
@@ -50,7 +50,8 @@ API Changes
 #. Removing the parameters ``outsideFuelRing`` and ``outsideFuelRingFluxFr``. (`PR#1700 <https://github.com/terrapower/armi/pull/1700>`_)
 #. Removing the setting ``doOrificedTH``. (`PR#1706 <https://github.com/terrapower/armi/pull/1706>`_)
 #. Changing the Doppler constant params to ``VOLUME_INTEGRATED``. (`PR#1659 <https://github.com/terrapower/armi/pull/1659>`_)
-#. Change ``Operator._expandCycleAndTimeNodeArgs`` to be a non-static method so it no longer needs to take arbitrary args (`PR#1766 <https://github.com/terrapower/armi/pull/1766>`_)
+#. Change ``Operator._expandCycleAndTimeNodeArgs`` to be a non-static method so it no longer needs to take arbitrary args. (`PR#1766 <https://github.com/terrapower/armi/pull/1766>`_)
+#. Database now writes state at the last time node of a cycle rather than during the `DatabaseInterface.interactEOC` interaction, making this node accessable since its the more useful node, and at a unique time. (`PR#1090 <https://github.com/terrapower/armi/pull/1090>`_)
```

Comment:
```suggestion
#. Database now writes state at the last time node of a cycle rather than during the `DatabaseInterface.interactEOC` interaction, making this node accessible since its the more useful node, and at a unique time. (`PR#1090 <https://github.com/terrapower/armi/pull/1090>`_)
```

Reply:
Do we also want to add another sentence here that says something like "This will break backwards compatibility with using old databases for restart calculations from the last node in a cycle."? Seems like it would be useful to have such important information in the release notes.

Reply:
done